scripts/dev-setup/main.py

```diff
@@ -19,6 +19,7 @@
     "promptflow-azure",
     "promptflow[azure]",
     "promptflow-tools",
+    # "promptflow-evals",
```

Comment:
any reason why commented?

Reply:
I was trying out something but have changed it back